code/modules/organs/organ.dm

```diff
@@ -290,9 +290,13 @@ var/list/organ_cache = list()
 	loc = get_turf(owner)
 	processing_objects |= src
 	rejecting = null
-	var/datum/reagent/blood/organ_blood = locate(/datum/reagent/blood) in reagents.reagent_list
-	if(!organ_blood || !organ_blood.data["blood_DNA"])
-		owner.vessel.trans_to(src, 5, 1, 1)
+	if (!reagents)
```

Comment:
it seems that severed limbs are given a tiny blood reservoir just for forensic sampling, to determine who they came from.

I'm not sure how it was failing to have a reagents set on it, but this fixes it